sqlvm-alwayson-cluster/README.md

```diff
@@ -46,3 +42,29 @@ This template creates the following resources:
 	+ 	Witness - Latest Windows Server 2012 R2 Image
 
 + 	Once deployed access can be gained by the public IP address of Primary Domain Controller
+
+## Deployment steps
+
+You can click the "deploy to Azure" button at the beginning of this document.
+
+|Name|Description|Example|
```

Comment:
@singhkay Can you provide an example of the table that will be created in Azure.com? what about users who view this in GitHub, they will not have an overview of the parameters?